akka-serialization-jackson/src/main/scala/akka/serialization/jackson/JacksonObjectMapperProvider.scala

```diff
@@ -246,13 +276,14 @@ class JacksonObjectMapperFactory {
   def newObjectMapper(@unused bindingName: String, jsonFactory: Option[JsonFactory]): ObjectMapper =
     new ObjectMapper(jsonFactory.orNull)
 
-  // FIXME Java API
-
   /**
    * After construction of the `ObjectMapper` the configured serialization features are applied to
    * the mapper. These features can be amended programatically by overriding this method and
    * return the features that are to be applied to the `ObjectMapper`.
    *
+   * When implementing a `JacksonObjectMapperFactory` with Java the `immutable.Seq` can be
+   * created with [[akka.japi.Util.immutableSeq]].
```

Comment:
I don't think custom `JacksonObjectMapperFactory` will be implemented by many so keeping it simple